### DISCUSSION TITLE
一些关于排版的想法

### DISCUSSION THREAD

**Discussion by 45gfg9** [yhwu-is/Linear-Algebra-I-H-for-ZJU-CKC#10] [Ideas]
- [x] 要不要给一些关键名词加上英文？效果类似这样：
```tex
则称$W$是$V$的\keyterm{线性子空间}{vector subspace}，简称\keyterm{子空间}{subspace}.
```
<img width="742" alt="截屏2023-07-05 00 01 52" src="https://github.com/yhwu-is/Linear-Algebra-for-ZJUCKC/assets/37043529/c702d09b-1c5b-45a7-aae3-479eb060000c">

这个可以作者们写的时候随便写，后续我来整理这些东西。

- [x] 例如 null / Ker、range / Im、span / L 等运算符要采用哪种表示？

- [x] ~~例如 𝐅 和 𝔽，选择哪种？我个人偏向于后者，空心体辨识度更高一些，相比来说更不容易引起歧义~~

- [x] 一些例如 `enumerate` 的环境中的编号可以再改进一下格式，这样就不用手动写编号了。这个我可以看看

**Comment by shadowash0215** [comment]
1. 工作量似乎有点大，可以大家一起的；
2. null, range; 
3. **F**
4. 您加油！

**Reply by 45gfg9** [reply]
1. 工作量无所谓的，因为其实没有那么多名词需要解释的（

**Comment by 45gfg9** [comment]
| 概念 | 代数与几何 | LADR | 采用 |
|------|------|---|--|
| 核空间 | ker | null | ker |
| 像空间 | im | range | im |
| 线性变换的矩阵 | 𝐌 | ℳ | 𝐌 |
| 多项式空间 | 𝐅\[x] | 𝒫(𝐅) | 𝐅\[x] |
| eigen- | 特征 | 本征 | 特征 |
| characteristic | | 特征 | 特征 |
| 张成空间 | L | span | span |
| 秩 | r | rank | r |
| 对偶空间 | V* | V' | V* |

**Comment by 45gfg9** [comment]
源文件的换行符怎么用？统一用 LF 吧？想加个 `.gitattributes`
